templates/pages/admin/assetdefinition/capacities.html.twig

```diff
@@ -162,6 +166,25 @@
                     $('div[data-capacity="' + $.escapeSelector($(this).data('capacity')) + '"]').toggleClass('d-none');
                 }
             });
+
+            // filter the list of capacities
+            let delay_timer = null;
+            $('#search-capacities').on('input', function() {
+                const search = $(this).val().toLowerCase();
+
+                // delay the search to avoid flickering
+                clearTimeout(delay_timer);
+                delay_timer = setTimeout(function() {
+                    $('input[name="capacities[]"]').each(function() {
+                        const capacity_search = $(this).data('capacity-search').toLowerCase();
+                        if (capacity_search.indexOf(search) === -1) {
+                            $(this).closest('.form-field').addClass('d-none');
+                        } else {
+                            $(this).closest('.form-field').removeClass('d-none');
+                        }
```

Comment:
There is a toggle function for this. Not a mandatory change.
```suggestion
                        $(this).closest('.form-field').toggleClass('d-none', capacity_search.indexOf(search) === -1);
```

Reply:
This is less readable no ?
I thought about doing this before pushing the pr, but it was clearer to make the if/else.